src/peer.js

```diff
@@ -60,13 +159,46 @@ function Peer(options) {
         // however we may want it in the future for broader device / browser support
         // offer.sdp = sdputils.maybeSetAudioReceiveBitRate(offer.sdp, self.streamConfig);
         // offer.sdp = sdputils.maybeSetVideoReceiveBitRate(offer.sdp, self.streamConfig);
+        let sdp = offer.sdp;
+        if (self.videoCodec) {
+            sdp = preferCodec(sdp, 'video', self.videoCodec);
+        }
+        if (self.audioCodec) {
+            sdp = preferCodec(sdp, 'audio', self.audioCodec);
+        }
+
+        if (self.videoSDPBitrate) {
+            sdp = setBitrate(sdp, 'video', '1024');
+        }
+        if (self.audioSDPBitrate) {
+            sdp = setBitrate(sdp, 'video', '1024');
+        }
+        offer.sdp = sdp;
+        console.log('offer sdp: ', offer.sdp);
         self.send('offer', offer);
     });
     this.pc.on('answer', function (answer) {
         if (self.parent.config.nick) answer.nick = self.parent.config.nick;
         // NOTE - this is to limit bitrate via SDP
         // answer.sdp = sdputils.maybeSetAudioReceiveBitRate(answer.sdp, self.streamConfig);
         // answer.sdp = sdputils.maybeSetVideoReceiveBitRate(answer.sdp, self.streamConfig);
+        let sdp = answer.sdp;
+        if (self.videoCodec) {
+            sdp = preferCodec(sdp, 'video', self.videoCodec);
+        }
+        if (self.audioCodec) {
+            sdp = preferCodec(sdp, 'audio', self.audioCodec);
+        }
+
+        if (self.videoSDPBitrate) {
+            sdp = setBitrate(sdp, 'video', self.videoSDPBitrate);
+        }
+        if (self.audioSDPBitrate) {
+            sdp = setBitrate(sdp, 'video', self.videoSDPBitrate);
+        }
+
+        answer.sdp = sdp;
+        console.log('answer sdp: ', answer.sdp);
```

Comment:
perhaps use `self.logger.log` ?

Reply:
oh i only added these for debugging w/ alex, i can just remove them

Reply:
:sweat_smile: